fairlearn/metrics/_disaggregated_result.py

```diff
@@ -183,7 +183,7 @@ def apply_grouping(
 
     def difference(
         self,
-        control_feature_names: List[str],
+        control_feature_names: Optional[List[str]],
```

Comment:
Shouldn't this have a default of `None`?
Also, it's missing a docstring section 😮

Reply:
Indeed :eyes: I added both (I actually copied the argument's docstring from `.apply_grouping`, which is the method that uses it inside `.difference` and `.ratio`)